src/script/user/UserRepository.ts

```diff
@@ -616,23 +613,19 @@ export class UserRepository {
 
   /**
    * Is the user the logged in user.
-   * @param {User|string} user_id - User entity or user ID
-   * @returns {boolean} Is the user the logged in user
    */
-  is_me(user_id) {
-    if (!_.isString(user_id)) {
+  is_me(user_id: User | string): boolean {
+    if (typeof user_id !== 'string') {
```

Comment:
Please keep the underscore check, it does more than just `typeof`. 

In this case a `new String` will return `false`.

```
typeof "test"; // => "string"
typeof new String("test"); // => object
```